identity/app/actions/AuthenticatedActions.scala

```diff
@@ -73,9 +93,14 @@ class AuthenticatedActions(
     }
   }
 
+
+  def authAction: AuthenticatedBuilder[AuthenticatedUser] =
+    new AuthenticatedBuilder(authService.authenticatedUserFor, anyContentParser, sendUserToSignin)
+
   def authActionWithUser: ActionBuilder[AuthRequest, AnyContent] =
-    authAction andThen apiVerifiedUserRefiner
+    DefaultActionBuilder(anyContentParser) andThen authRefiner andThen apiVerifiedUserRefiner
```

Comment:
I know I suggested this `DefaultActionBuilder` usage but I think it would still be preferable to kick off this chain with an `ActionBuilder` that does something.  Looking at the other classes interacting with `DefaultActionBuilder`, I think something like this might work (ignore the naming of things, which would need changing):

```
class AuthActionBuilder[B](implicit val executionContext: ExecutionContext)
    extends DefaultActionBuilderImpl(anyContentParser) {
  override def invokeBlock[A](request: Request[A], block: (Request[A]) => Future[Result]) = doSomething(request)
}
```

Where `doSomething` is roughly equivalent to `refine` right now.

Reply:
Then presumably you'd have (pseudo-code):

```
object AuthActionBuilder {
  def apply() = new AuthActionBuilder()
}

def authActionWithUser: ActionBuilder[AuthRequest, AnyContent] =
  AuthActionBuilder andThen authRefiner andThen apiVerifiedUserRefiner